src/operator/tensor/matrix_op-inl.h

```diff
@@ -17,7 +17,7 @@ namespace op {
 struct ReshapeParam : public dmlc::Parameter<ReshapeParam> {
   TShape target_shape;
   bool keep_highest;
-  nnvm::Tuple<int> shape;
+  TShape shape;
```

Comment:
Then I'd like to `DMLC_DECLARE_TYPE_NAME` for it, since this info is missing when using `MXSymbolGetAtomicSymbolInfo`, where do you think can I put the declartion?